WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -654,6 +655,8 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
                 mediaType = mediaType | WPMediaType.other.rawValue
             case .any:
                 mediaType = mediaType | WPMediaType.all.rawValue
+            @unknown default:
+                fatalError()
```

Comment:
For some reason, when building the app using a local installation of Gutenberg, this line (and similar ones below) are giving the following warning when building: **Default will never be executed**

However, this is not the case when building using the XCFramework 🤔